recipes-demos/qmltermwidget/qmltermwidget/0001-qmltermwidget.pro-don-t-install-asset-directories-tw.patch

```diff
@@ -0,0 +1,44 @@
+From df920277cab106584e04a3704d66e49251c731d5 Mon Sep 17 00:00:00 2001
```

Comment:
What's the status of the patch getting merged in qmltermwidget sources?

Reply:
I tried without building the patch. The recipe fails if we don't apply the patch.
As we are picking an older version qt5 from the layer ( and also from the qmltermwidget project), this patch wont be there for qt5. Both qmltermwidget and the layer supports qt6 now and it might work without any patches if also use qt6. So either we somehow fix this in yocto later or keep the patch for building this recipe.